collie/model/base/trainer.py

```diff
@@ -176,12 +176,21 @@ class CollieMinimalTrainer():
     max_depth: int
         Maximum depth of modules to show, use -1 to show all modules or 0 to show no summary.
         Related to PyTorch Lightning's ``Trainer.enable_model_summary`` argument.
+    enable_model_summary: bool
+        Whether to enable or disable the model summarization. Defaults to True.
     weights_summary: str
         Deprecated, replaced with ``max_depth``. Prints summary of the weights when training begins
     detect_anomaly: bool
-        Enable anomaly detection for the autograd engine.
+        Context-manager that enable anomaly detection for the autograd engine. This does two things:
+
+        * Running the forward pass with detection enabled will allow the backward pass to print the
+          traceback of the forward operation that created the failing backward function.
+
+        * Any backward computation that generate “nan” value will raise an error.
+        Warning: This mode should be enabled only for debugging as the different tests will slow
+        down your program execution.
```

Comment:
This is one of those weird formatting edge-cases where a bullet-point list has to be formatted specifically to render properly with the docs. Basically, they need to have an extra tab applied to them and have end-spaces at the end. 
```suggestion
            * Running the forward pass with detection enabled will allow the backward pass to print the
              traceback of the forward operation that created the failing backward function.

            * Any backward computation that generate “nan” value will raise an error.
            Warning: This mode should be enabled only for debugging as the different tests will slow
            down your program execution.

```

See here for an example of this: https://github.com/ShopRunner/collie/blob/main/collie/model/base/base_pipeline.py#L61

Reply:
Thank you!